src/Algebra/Consequences/Base.agda

```diff
@@ -14,9 +14,19 @@ open import Algebra.Core
 open import Algebra.Definitions
 open import Data.Sum.Base
 open import Relation.Binary.Core
+open import Relation.Binary.Definitions using (Reflexive)
+
+module _ {ℓ} {_•_ : Op₂ A} (_≈_ : Rel A ℓ) where
+
+  sel⇒idem : Selective _≈_ _•_ → Idempotent _≈_ _•_
+  sel⇒idem sel x with sel x x
+  ... | inj₁ x•x≈x = x•x≈x
+  ... | inj₂ x•x≈x = x•x≈x
+
+module _ {ℓ} {f : Op₁ A} (_≈_ : Rel A ℓ) where
+
+  reflexive+selfinverse⇒involutive : Reflexive _≈_ →
```

Comment:
In names it should be referred to as `selfInverse` everywhere rather than the current mix of `selfinverse` and `self-inverse`